### ISSUE TITLE
New module release

### ISSUE THREAD

**Issue by AntonKarasov** [openresty/lua-nginx-module#2342]
Hello,

Are you going to release a new module version?

https://github.com/openresty/lua-nginx-module/blob/v0.10.26/t/044-req-body.t#L1796
https://github.com/openresty/lua-nginx-module/pull/2237/files#diff-822bb33b8ca01faca3dc8070342892535fd34453c6637b69fe3d30f228d22bbcR141
This limitation kind of blocks us and it seems like it was already revered/fixed within the master branch but there have been no new module releases for a while...

I wonder when you are going to do it since finding some workarounds to be able to beat such limitation when we need request body is kinda sucks...

Apart from that if you have any tips on how to handle it on the configs side - I'd really appreciate it.

Basically, we are using Openresty as a proxy that is able to write request and response bodies to access logs for our further analytics and faced with such a 500 status code when there is no content-length header in the request...

Thanks and looking forward to your reply.

**Comment by spacewander** [comment]
AFAIK, @zhuizhuhaomeng plans to release a new version that matches Nginx 1.27.

**Comment by spacewander** [comment]
Solved by https://github.com/openresty/lua-nginx-module/releases/tag/v0.10.27
